### PR TITLE
Add weekly quizzes to the jssandbox

### PR DESCRIPTION
**Pull request by imsickofmaps**
Should implement the display of Issue #19

### CODE REVIEW

**Comment by imsickofmaps** [comment]
@smn ready for review but easier after landing #8 #21 #22 and #23 to clear merges from there

File: js_sandbox/lib/go-nike-ghr.js [hunk 1]
```diff
@@ -404,6 +584,14 @@ function GoNikeGHR() {
             }
             return self.error_state();
         });
+        p_mandl.add_callback(function(){
+            var p_opinion = self.crm_get('opinion/');
+            p_opinion.add_callback(function(result){
+                im.config.opinions = result.opinions;
```
Comment (smn) [inline js_sandbox/lib/go-nike-ghr.js:535]: Can you explain what's going on here?

**Comment by smn** [comment]
This branch will need some work as there's an assumption that states will always exist after being created. The only "thing" that's guaranteed to be run (other than the current & next state) is the config load stuff. Any dynamic state that need to exist should be created there before the actual "current_state" is handled.

**Comment by imsickofmaps** [comment]
@smn I've moved all the generators across the features into on_config_load now. Shall I merge?

**Comment by smn** [comment]
:+1: on the changes

**Pull request merged**